portal/exercise_diet/templates/exercise_diet/diet.html

```diff
@@ -31,17 +36,14 @@
                         {% endif %}
                     </div>
                     {% endif %}
-                    
+
                     <div class="modal-footer">
                         {% trans %}
                         <button type="button" class="btn btn-secondary" data-dismiss="modal"><b>close</b><span>X</span></button>
```

Comment:
Please wrap button with translation tags

Reply:
Hmm, I think that one is wrapped already

Reply:
Whoops, sorry